doc/source/configuration.rst

```diff
@@ -3,115 +3,258 @@
 Configuration
 =============
 
-You are looking at a highly configurable piece of software. To get the right understanding of the server it
+You are looking at a highly configurable piece of software. To get a good understanding of the server it
 is recommended to read this part carefully.
 
 Since the swiss confederation's definition and the specification for the `OEREB Data Extract
```

Comment:
Maybe: The specifications of the oereb data model, the web services, the data extracts and the print layout by the Swiss Confederation are very precise (https://www.cadastre.ch/de/manual-oereb/publication/instruction.html). Code development was guided by the exact implementation of the specifications and the user requirements. Using this pyramid plugin, you will get a running server providing all the services defined by the federal specifications. The binding of cantonal and federal data to the server is done by the configuration options.